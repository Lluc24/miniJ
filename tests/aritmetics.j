1 2 3 + _2 _2 3              NB. resultat: _1 0 6
_7 4 0 - 1 2 _3              NB. resultat: _8 2 3
3 0 _7 2 * _3 4 _1 2         NB. resultat: _9 0 7 4
_8 _2 7 0 % _3 2 _3 4        NB. resultat: 2 _1 _3 0
_3 0 _1 2 ^ 3 7 5 10         NB. resultat: _27 0 _1 1024
5 _3 _2 3 | 11 8 _5 _5       NB. resultat: 1 _1 _1 1

2 ^ 2 ^ 3                    NB. resultat: 256
(5 * 2 + 1) % 4 - 1          NB. resultat: 5

*: 3                         NB. resultat: 9
- / 1 2 3                    NB. resultat: 2
^ / 2 2 3                    NB. resultat: 256
