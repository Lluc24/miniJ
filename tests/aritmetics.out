_1 0 6
_8 2 3
_9 0 7 4
2 _1 _3 0
_27 0 _1 1024
1 _1 _1 1
256
5
9
2
256
